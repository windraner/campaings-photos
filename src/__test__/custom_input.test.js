import React from 'react';
import CustomInput from '../components/filters_container/custom_input/custom_input';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

const mock = {
  searchInput: '',
  stateHandler() {},
};

describe('CustomInput', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<CustomInput {...mock} />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
