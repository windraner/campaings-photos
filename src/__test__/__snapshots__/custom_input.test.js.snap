// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput matches the snapshot 1`] = `
<div
  className="position-relative"
>
  <input
    className="campaigns-photos__filter-input"
    onBlur={[Function]}
    onChange={[Function]}
    placeholder="Search list"
    type="text"
    value=""
  />
  <span
    className="campaigns-photos__filter-input-icon d-flex align-items-center"
  />
</div>
`;
